install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -101,7 +101,7 @@ spec:
               command:
                 - /usr/local/bin/galley
                 - probe
-                - --probe-path=/healthready
+                - --probe-path=/tmp/healthready
```

Comment:
Could you also add an emptyDir at `/tmp` this would allow galley to run with `readOnlyRootFilesystem`?